transformation/src/jade-schema/tables/biosample.table.json

```diff
@@ -0,0 +1,147 @@
+{
+  "name": "biosample",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "source",
+      "datatype": "string"
+    },
+    {
+      "name": "data_use_limitations",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "data_use_requirements",
```

Comment:
I believe all data use stuff is constant across ENCODE. If so, I think we can leave out these columns.

Reply:
(There's no machinery in the repo to drive access based on this, and when it goes in I'm not sure it'll be driven by row-level primary data vs. some other mechanism).

Reply:
👍 I just removed the `data_use_requirements` and `data_use_limitations` columns.